client/components/open/components/RecordOperations.vue

```diff
@@ -100,17 +101,23 @@ const onDeleteClick = () => {
 }
 
 const deleteRecord = () => {
-  formsApi.submissions.delete(props.form.id, submission.value.id)
-    .then((data) => {
-      if (data.type === "success") {
-        emit("deleted", submission.value)
-        alert.success(data.message)
-      } else {
-        alert.error("Something went wrong!")
+  deleteSubmissionMutation.mutate(
+    { 
+      formId: props.form.id, 
+      submissionId: submission.value.id 
+    },
+    {
+      onSuccess: (data) => {
```

Comment:
Same do not use this but instead on vue file when triggering a function use mutate async. If there's a form you can also directly use it to submit like this: 

https://github.com/JhumanJ/OpnForm/blob/62d013f5ee5337a4a396bb39a9034e7ebe45413c/client/composables/lib/vForm/Form.js#L221-L229